Core/MIPS/ARM64/Arm64CompVFPU.cpp

```diff
@@ -221,23 +221,17 @@ namespace MIPSComp {
 			// CC might be set by slow path below, so load regs first.
 			fpr.MapRegV(vt, MAP_DIRTY | MAP_NOINIT);
 			if (gpr.IsImm(rs)) {
-				u32 addr = offset + gpr.GetImm(rs);
-				gpr.SetRegImm(SCRATCH1_64, addr + (uintptr_t)Memory::base);
+				u32 addr = (offset + gpr.GetImm(rs)) & 0x3FFFFFFF;
```

Comment:
And this?

Reply:
Both removed, plus others.

-[Unknown]